pkgs/tools/compression/dtrx/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchFromGitHub, python3Packages
+, gnutar, unzip, lhasa, rpm, binutils, cpio, gzip, p7zip, cabextract, unrar, unshield
+, bzip2, xz, lzip
+# unzip is handled by p7zip
+, unzipSupport ? false
+, unrarSupport ? false }:
+
+let
+  archivers = lib.makeBinPath ([ gnutar lhasa rpm binutils cpio gzip p7zip cabextract unshield ]
+  ++ lib.optional (unzipSupport) unzip
+  ++ lib.optional (unrarSupport) unrar
+  ++ [ bzip2 xz lzip ]);
```

Comment:
Why these three are isolated from the others?

Reply:
I reverted the commit which removed dtrx. I have no idea!